src/qt/buynamespage.cpp

```diff
@@ -45,22 +49,61 @@ bool BuyNamesPage::eventFilter(QObject *object, QEvent *event)
 {
     if (event->type() == QEvent::FocusIn)
     {
-        if (object == ui->registerName)
+        if (object == ui->registerNameAscii)
+        {
+            ui->registerNameButton->setDefault(true);
+        }
+
+        if (object == ui->registerNameHex)
+        {
+            ui->registerNameButton->setDefault(true);
+        }
+
+        if (object == ui->registerNameDomain)
         {
             ui->registerNameButton->setDefault(true);
         }
+
     }
     return QWidget::eventFilter(object, event);
 }
 
-void BuyNamesPage::onNameEdited(const QString &name)
-{
-    if (!walletModel)
-        return;
 
-    const QString availableError = name_available(name);
+QString BuyNamesPage::DomainToASCII(const QString &name){
+    
+    if(IsPurportedNamecoinDomain(name.toStdString()))
+    {
+        return QString::fromStdString(ASCIIFromDomain(name.toStdString()));
+    }
+    else return QString("");
```

Comment:
`else` isn't needed here, since the `then` clause of this `if` statement will always `return` before we reach this point. Please remove `else` and add a blank line between the `if` statement and the `return`.

Reply:
The change here isn't correct -- `return` is still needed, just not `else`.